pkg/services/object_manager/transformer/types.go

```diff
@@ -0,0 +1,90 @@
+package transformer
+
+import (
+	"io"
+
+	objectSDK "github.com/nspcc-dev/neofs-api-go/pkg/object"
+	"github.com/nspcc-dev/neofs-node/pkg/core/object"
+)
+
+// AccessIdentifiers represents group of the object identifiers
+// that are returned after writing the object.
+// Consists of the ID of the stored object and the ID of the parent object.
+type AccessIdentifiers struct {
+	par, self *objectSDK.ID
+}
+
+// ObjectTarget is an interface of the object writer.
+type ObjectTarget interface {
+	// WriteHeader writes object header w/ payload part.
+	// The payload of the object may be incomplete.
+	//
+	// Must be called exactly once. Control remains with the caller.
+	// Missing a call or re-calling can lead to undefined behavior
+	// that depends on the implementation.
+	//
+	// Must not be called after Close call.
+	WriteHeader(*object.RawObject) error
+
+	// Write writes object payload chunk.
+	//
+	// Can be called multiple times. Must not be called
```

Comment:
Remove `Must not be called`, explained in next line.

Reply:
Fixed